azure-kusto-data/azure/kusto/data/__init__.py

```diff
@@ -1,4 +1,148 @@
-from .kusto_client import KustoClient, KustoResultIter, KustoResponse
-from .kusto_exceptions import KustoError, KustoClientError, KustoServiceError
+from enum import Enum
+from six import text_type
 from .version import VERSION
 __version__ = VERSION
+
+class KustoConnectionStringBuilder(dict):
```

Comment:
Why are you subclassing dict? Any reason you can't simply work with a dict? Or just a normal class?

Reply:
KustoConnectionStringBuilder meant to be a dictionary that has limited keys. If I don't subclass dict I need to write this functionality by myself, of do you have other idea How can this be done?